### DISCUSSION TITLE
一份简明的 Base64 原理解析 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#244] [Announcements]
https://mazhuang.org/2020/03/07/get-an-in-depth-understanding-of-base64/ 

 书接上回，在 记一个 Base64 有关的 Bug 一文里，我们说到了 Base64 的编解码器有不同实现，交叉使用它们可能引发的问题等等。

**Comment by ziyu123** [comment]
dd
